test/parallel/test-tls-lookup.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const tls = require('tls');
+
+const expectedError = /^TypeError: "lookup" option should be a function$/;
+
+['foobar', 1, {}, []].forEach(function connectThrows(input) {
+  const opts = {
+    host: 'localhost',
+    port: common.PORT,
```

Comment:
There is an ongoing effort to avoid the usage of `common.PORT`. cc @Trott Can you please change the port number to zero, if it is applicable?